test/functional/wallet_multiwallet.py

```diff
@@ -151,7 +151,9 @@ def run_test(self):
         assert_equal(w3.getbalance(), 0)
         assert_equal(w4.getbalance(), 0)
 
-        w1.sendtoaddress(w2.getnewaddress(), 1)
+        w2addrR = w2.getnewaddress()
```

Comment:
nit: I'm not sure what the `R` is for here. In general, prefer to use `lower_case_with_underscores` for [variable names in Python](https://www.python.org/dev/peps/pep-0008/#id45).